### PR TITLE
Added some rules to verify assembly attributes

### PR DESCRIPTION
**Pull request by ajaybhargavb**
Issue #4 

@Eilon / @pranavkm

### CODE REVIEW

File: src/NuGetPackageVerifier/Program.cs [hunk 1]
```diff
@@ -68,6 +68,12 @@ public static int Main(string[] args)
 
             // TODO: Look this up using reflection or something
             var allRules = new IPackageVerifierRule[] {
+                new AssemblyHasCompanyAttributeRule(),
+                new AssemblyHasCopyrightAttributeRule(),
+                new AssemblyHasDefaultAliasAttributeRule(),
+                new AssemblyHasDescriptionAttributeRule(),
+                new AssemblyHasProductAttributeRule(),
+                new AssemblyHasTitleAttributeRule(),
```
Comment (Eilon) [inline src/NuGetPackageVerifier/Program.cs]: Uh oh not sorted anymore :smile:

File: src/NuGetPackageVerifier/Rules/AssemblyHasAttributeRuleBase.cs [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+                            File.Delete(assemblyPath);
+                        }
+                    }
+                }
+            }
+
+            return Enumerable.Empty<PackageVerifierIssue>();
+        }
+
+        public virtual IEnumerable<PackageVerifierIssue> ValidateAttribute(
```
Comment (Eilon) [inline src/NuGetPackageVerifier/Rules/AssemblyHasAttributeRuleBase.cs]: Make abstract?

File: src/NuGetPackageVerifier/CompositeRules/DefaultCompositeRule.cs [hunk 3]
```diff
@@ -11,6 +11,12 @@ public class DefaultCompositeRule : IPackageVerifierRule
     {
         IPackageVerifierRule[] _rules = new IPackageVerifierRule[]
         {
+            new AssemblyHasCompanyAttributeRule(),
+            new AssemblyHasCopyrightAttributeRule(),
+            new AssemblyHasDefaultAliasAttributeRule(),
```
Comment (Eilon) [inline src/NuGetPackageVerifier/CompositeRules/DefaultCompositeRule.cs]: Hmm reading MSDN I don't think we need DefaultAlias. It sounds like some COM thing or something. Can probably get rid of it.

**Comment by Eilon** [comment]
:watch:

**Comment by ajaybhargavb** [comment]
:up: :date:

File: src/NuGetPackageVerifier/CompositeRules/DefaultCompositeRule.cs [hunk 4]
```diff
@@ -11,6 +11,11 @@ public class DefaultCompositeRule : IPackageVerifierRule
     {
         IPackageVerifierRule[] _rules = new IPackageVerifierRule[]
         {
+            new AssemblyHasCompanyAttributeRule(),
+            new AssemblyHasCopyrightAttributeRule(),
+            new AssemblyHasDescriptionAttributeRule(),
+            new AssemblyHasProductAttributeRule(),
+            new AssemblyHasTitleAttributeRule(),
```
Comment (Eilon) [inline src/NuGetPackageVerifier/CompositeRules/DefaultCompositeRule.cs]: Still not sorted?

**Comment by Eilon** [comment]
:shipit:

File: src/NuGetPackageVerifier/CompositeRules/DefaultCompositeRule.cs (hunk 4 above)
Comment (ajaybhargavb) [inline src/NuGetPackageVerifier/CompositeRules/DefaultCompositeRule.cs]: Oops. Sorted in `Program.cs`. Missed this.

Comment (Eilon) [inline src/NuGetPackageVerifier/CompositeRules/DefaultCompositeRule.cs]: Ah yes it's a different place, I didn't even notice that myself.

**Pull request merged**